packages/react-native-skia/modules/Platform/platform.js

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View,
+  Platform,
+  TouchableHighlight
+} from "react-native";
+
+const SimpleViewApp = () => {
+  const [responseText, setResponseText] = React.useState();
+  const GetSystemName = () => {
+    let responseString =
+    "Platform OS:" +Platform.OS+"\n" +
+    "Platform Os Version():" +Platform.Version +"\n" +
+    "Platform IsTv():" +Platform.isTV +"\n" +
+    "Platform IsPad():" +Platform.isPad +"\n" +
+    "Platform Testing():" +Platform.isTesting +"\n" +
+    "Platform Constants():" +Platform.constants + "\n"+
+    "Platform Constants System Name:" +Platform.constants.systemName +"\n" +
+    "Platform Constant forceTouchAvailable:"+Platform.constants.forceTouchAvailable +"\n" +
+    "Platform Constant reactNativeVersion:" +Platform.constants.reactNativeVersion.major+"." 
+                                            +Platform.constants.reactNativeVersion.minor+"." 
+                                            +Platform.constants.reactNativeVersion.patch+"-"
+                                            +Platform.constants.reactNativeVersion.prerelease+ "\n"+
+    "Platform Constant interfaceIdiom:" +Platform.constants.interfaceIdiom +"\n" +
+    "Platform Constant Os Version:" +Platform.constants.osVersion +"\n" +
+    "Platform Constant is Testing:" +Platform.constants.isTesting +"\n" +
+    "Platform Select:" +Platform.select +"\n" 
+   
+      setResponseText(responseString);
+    
+  }; 
+
+  return (
+    <View style={styles.mainView}>
+      <View>
+        <TouchableHighlight
+          isTVSelectable="true"
```

Comment:
What is "isTVSelectable" supposed to do.
It could get reference of it in ReactNative documentation, do I need to refer any specific version ?

Reply:
isTVSelectable  (TVOS) when set to true, this view will be focusable and navigable using the TV remote. This may not be needed for Touchable components as they might be already having this internally( in RN TVOS)